src/api.ts

```diff
@@ -162,8 +164,10 @@ export class Api extends Connection implements ApiInterface {
                     this.saveTokens(this.authData);
                     planRefresh();
                 } catch (e) {
-                    // remove tokens from storage if expired
-                    this.removeTokens();
+                    if (e instanceof HttpTokenExpiredError) {
+                        // remove tokens from storage if expired
+                        this.removeTokens();
+                    }
                     console.error('Token autoRefresh failed');
```

Comment:
mb need to planRefresh if we got error that token not expired?

Reply:
added refresh attempts